static/help-docs/authors.md

```diff
@@ -1,86 +1,6 @@
-
-5. Authenticate to GitHub by running `pulp login`. (This is necessary in order for us to be able to tell who uploaded which packages).
-
-6. Change to your project directory and run `pulp publish`. This will the push commits and the relevant tag to your "origin" Git remote, and then generate your documentation and upload it to Pursuit.
-
-    `pulp publish` also accepts a `--no-push` flag which skips the Bower registration check as well as pushing commits (this is useful for uploading other people's packages, if you ever need to do this). There is also a `--push-to` option which allows you to specify a different Git remote to push tags and commits to.
-
-    **Note: If `pulp publish` fails with a `400` error, try running it a second time.** Usually, your project's documentation will be successfully published on the second run. For example:
-
-    ```
-    # tag gets pushed in first run, so we don't need it in the second run
-    yes | pulp publish
-    yes | pulp publish --no-push
-    ```
-
-
-Your package, together with documentation, should now appear in Pursuit.
-
-## <a name="submit-automated"></a>Submitting packages from a script
-
-You can also use Pulp to submit packages from a script. Pulp prompts for confirmation when you run `pulp publish`, so you will need to use a program like `yes` to answer affirmatively.
-
-For example, if you want to automatically upload documentation from your Travis CI build on tags, you should add the following to your `after_script` build step:
-
-`test -n "$TRAVIS_TAG" && ( yes | pulp publish --no-push )`
-
-Alternatively, if you don't want to use Pulp for whatever reason, you can upload packages using Pursuit's HTTP API directly:
-
-- Gzip the JSON output produced by `psc-publish` and save it to a file.
-- If you don't already have one, get a GitHub API token by visiting <https://github.com/settings/tokens/new>. No scopes are required, since the token is only used for authentication.
-- Make a POST request to https://pursuit.purescript.org/packages, with gzipped JSON as the request body, including a `Content-Encoding: gzip` header, and with your GitHub token in the Authorization header, like this: `Authorization: token {token}`.
-
-For example, using curl:
-
-```
-curl -X POST \
-  https://pursuit.purescript.org/packages \
-  --data-binary @pursuit.json.gz \
-  -H 'Content-Encoding: gzip' \
-  -H 'Accept: application/json' \
-  -H "Authorization: token $(cat my-oauth-token.txt)" \
-  -v
-```
-
-If your submission is successful, Pursuit will return a 201 Created response, and the URL for your newly uploaded package will be in the Location header.
+Previously, users would need to use `bower` to upload packages' documentation. Uploading documentation is now handled by the PureScript Registry (still in alpha at the time of this writing). See https://discourse.purescript.org/t/registry-alpha-launched/3146 and any updates mentioned there for how to register and publish packages and their documentation.
```

Comment:
This should probably link directly to the registry GitHub readme, rather than the discourse link — or perhaps to both?